src/test/java/org/openelisglobal/analysisqaeventaction/AnalysisQaEventActionServiceTest.java

```diff
@@ -0,0 +1,190 @@
+package org.openelisglobal.analysisqaeventaction;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.analysisqaeventaction.service.AnalysisQaEventActionService;
+import org.openelisglobal.analysisqaeventaction.valueholder.AnalysisQaEventAction;
+import org.openelisglobal.common.util.ConfigurationProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class AnalysisQaEventActionServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private AnalysisQaEventActionService analysisQaEventService;
+
+    private List<AnalysisQaEventAction> analysisQaEventActions;
+    private Map<String, Object> propertyValues;
+    private List<String> orderProperties;
+    private static int NUMBER_OF_PAGES = 1;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSetWithStateManagement("testdata/analysis-qa-event-action.xml");
+        propertyValues = new HashMap<>();
+        orderProperties = new ArrayList<>();
+    }
+
+    @Test
+    public void testGetAllEventActions() {
```

Comment:
the naminc convension should be ` getAll_shouldGetAllEventActions` ie `methodName_expectedBehaviour`